nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc

```diff
@@ -0,0 +1,471 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * NNStreamer tensor_trainer subplugin for nntrainer
+ * Copyright (C) 2022 Hyunil Park <hyunil46.park@samsung.com>
+ */
+/**
+ * @file   tensor_trainer_nntrainer.cc
+ * @date   02 Dec 2022
+ * @brief  NNStreamer tensor_trainer subplugin
+ * @see    http://github.com/nnstreamer/nnstreamer
```

Comment:
```suggestion
 * @see    http://github.com/nnstreamer/nntrainer
```

Reply:
Thank you. Because it is a nnstreamer sub-plugin, It was told to look at 'nnstreamer'
thank you.

Reply:
Then... is it Apache-2.0 license?? Could you check the license?
and my thinking is how about add two? nnstreamer and nntrainer?

Reply:
It is same with tensor_filter_nntrainer.cc
```
/* SPDX-License-Identifier: Apache-2.0 */
/**
 * NNStreamer Tensor_Filter, nntrainer Module
 * Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
 */
/**
 * @file   tensor_filter_nntrainer.cc
 * @date   09 Sept 2020
 * @brief  nntrainer inference module for tensor_filter gstreamer plugin
 * @see    http://github.com/nnstreamer/nnstreamer
 * @author Jijoong Moon <jijoong.moon@samsung.com>
 * @bug    No known bugs except for NYI items
 *
 * This is the per-NN-framework plugin (e.g) tensorflow-lite) for tensor_filter.
 * Fill in "GstTensorFilterFramework" for tensor_filter.h/c
 *
 */
 ```

Reply:
License  Apache-2.0 is correct. thank you.